network.env.dist

```diff
@@ -126,3 +126,4 @@ ENABLE_PPROF=false
 # Output format variables
 SHOW_COLORS=true
 RUN_BLAKE3=false
+DB_ENGINE=pebble
```

Comment:
Should this default to level still, until we are ready to make pebble the default?

Reply:
Dont we want to run pebble as default and if we see any issues go back to pebble?